### PR TITLE
Optimize lodash imports

### PR DESCRIPTION
**Pull request by ascariandrea**
To decrease the build size I've only imported `lodash` real used modules.
If this can help I can do this also for yours others modules.

Thanks

### CODE REVIEW

**Comment by knowbody** [comment]
this is a good idea, thank you @ascariandrea. But as we use lodash in quite a few places [babel-plugin-lodash](https://github.com/megawac/babel-plugin-lodash) might be something what we could add to the builder config. cc @ryan-roemer @boygirl

**Comment by ascariandrea** [comment]
I didn't know this plugin exists! I'll give it a look also for my projects ;)

**Comment by boygirl** [comment]
`babel-plugin-lodash` sounds like an easy win!

**Comment by boygirl** [comment]
actually @ascariandrea I like your approach more than using a babel plugin. I'll double check this PR and we can work on getting it in.

File: package.json [hunk 1]
```diff
@@ -23,6 +23,7 @@
     "builder": "~2.1.4",
     "builder-victory-component": "~0.0.5",
     "d3-scale": "^0.3.0",
+    "lodash": "^3.10.1",
```
Comment (boygirl) [inline package.json]: This isn't necessary as it is already included via the `builder-victory-component` archetype. Please remove.

**Comment by boygirl** [comment]
@ascariandrea this is approved pending the change to `package.json` mentioned above. Thanks for lending a hand!

File: package.json (hunk 1 above)
Comment (ascariandrea) [inline package.json]: I'm going to remove that, but I think the `dependencies` should be declared in the current project `package.json` even if they are common for multiple project. What do you think?

**Comment by ascariandrea** [comment]
@boygirl Thanks to you!

**Comment by boygirl** [comment]
@ascariandrea I'll open a discussion issue about package vs. archetype depedencies

**Comment by ascariandrea** [comment]
@boygirl mention me if you want.
Can I optimize `lodash` imports in other projects?

**Comment by boygirl** [comment]
@ascariandrea yes, please do!

**Pull request merged**